tools/wptrunner/wptrunner/formatters/chromium.py

```diff
@@ -125,30 +125,34 @@ def _get_expected_status_from_data(self, actual_status, data):
         is expected. So we use the "expected" status if it's there or fall back
         to the actual status if it's not.
 
+        If the test has multiple statuses, it will have other statuses listed as
+        "known_intermittent" in |data|. If these exist, they will be appended to
+        the returned status with spaced in between.
+
         :param str actual_status: the actual status of the test
         :param data: a data dictionary to extract expected status from
-        :return str: the expected status
+        :return str: the expected statuses as a string
         """
-        return (self._map_status_name(data["expected"])
-                if "expected" in data else actual_status)
+        expected_statuses = self._map_status_name(data["expected"]) if "expected" in data else actual_status
+        if "known_intermittent" in data:
+            expected_statuses += " " + " ".join(
+                [self._map_status_name(other_status) for other_status in data["known_intermittent"]])
+        return expected_statuses
 
     def suite_start(self, data):
         self.start_timestamp_seconds = (data["time"] if "time" in data
                                         else time.time())
 
     def test_status(self, data):
         test_name = data["test"]
-        is_unexpected = None
         actual_status = self._map_status_name(data["status"])
-        expected_status = self._get_expected_status_from_data(actual_status, data)
+        expected_statuses = self._get_expected_status_from_data(actual_status, data)
 
-        is_unexpected = actual_status != expected_status
+        is_unexpected = actual_status not in expected_statuses
```

Comment:
Would it be possible if `actual_status` is empty or space?

Reply:
No, actual should always be something. If it were to happen it'd be a bug and there's no great graceful recovery from that - so this is as good as anything (outputting an empty "actual" and marking it as unexpected).